### DISCUSSION TITLE
Polar Alignment correction

### DISCUSSION THREAD

**Discussion by fenriques** [mworion/MountWizzard4#228] [Feature Request]
hi Michel,

on Ekos there's a nice feature for PA correction, maybe you're already familiar with that. 
After the first iteration on the PA routine (e.g. on three stars) the software is able to calculate the PA error and direction. 
Ekos then starts a live imaging session (1 frame every n secs) and allows you to move the RA and DEC axis to the final correct position.
As a visual aid it draws on the imaging window a reference vector that acts like a guide for the direction and magnitude of the correction.
Would it be feasible on MW?

Ferrante

**Comment by mworion** [comment]
Hi Ferrante,
actually I use the PA routing build in in the mount computer. You need to build a small model (which ist quite fast) and then select any reference star in hemisphere window. Afterwards you only have to center this star to the cross hair. No need to see Polaris. 
I would normally avoid building an existing feature a second time.
For what reason do you need that in MW4 ? Could you explain a little bit more how you do your PA normally. Perhaps I could tweak the process a little bit.
Michel

Edit: If you don't mind I shift it to discussion.

**Comment by fenriques** [comment]
hi Michel,
by what you are telling me, it seems MW already has this feature! 
I simply wasn't aware of it and will give it a try tonight.

Anyway what I meant is shown in this video starting at 4:20 : https://www.youtube.com/watch?v=5btq7an9mUE 
But again, seems to be not a new feature for MW.

MW is getting so feature rich that I struggle to keep track of all the improvements. 
And I take the chance to thank you very much for giving us such a useful tool.

Ferrante

**Reply by fenriques** [reply]
ok I made some tests, what I meant was a process that follows the standard PA routine: after the three (at least) plate solve solution are found, Ekos gives a visual hint on the PA correction. 
It starts taking frames every second and guides you in moving the RA/DEC knobs to the correct polar position.

ferrante

**Reply by mads0100** [reply]
The MW4 routine lets you do something similar; while it doesn't give you a vector to the star as you're describing and updates on your polar alignment on the way.... it does let you center a star up.  I find, with the right model, it's better than using the Ekos method.  Both are OK; you don't get as much 'feedback' with the MW4/10u, but you don't really need it either.
